plugin.php

```diff
@@ -496,57 +485,57 @@ function dequeue_styles() {
 	}
 
 
-	function filter_ajax_echo( $items, $result ) {
-
-		$form = WPCF7_ContactForm::get_current();
-		$track_ga_submit = $this->get_form_settings( $form, 'track-ga-submit' );
-
-		if ( ! empty( $track_ga_submit ) ) {
-
-			if ( ! isset( $items['onSubmit'] ) )
-				$items['onSubmit'] = array();
-
-			$items['onSubmit'][] = sprintf(
-					'if ( typeof ga == "function" ) {
-						ga( "send", "event", "Contact Form", "Submit", "%1$s" );
-					}
-					if ( typeof _gaq !== "undefined" ) {
-						_gaq.push([ "_trackEvent", "Contact Form", "Submit", "%1$s" ]);
-					}',
-					esc_js( $form->title() )
-				);
+	function track_form_events() {
```

Comment:
Collect a list of forms that will trigger analytics events and/or redirects and make them available to our new JS.